packages/main/src/MultiComboBoxTemplate.tsx

```diff
@@ -0,0 +1,118 @@
+
+			{this.hasValueState &&
+				<span id="ui5-multi-combobox-valueStateDesc" class="ui5-hidden-text">{this.ariaValueStateHiddenText}</span>
+			}
+
+			<Tokenizer
+				slot="_beginContent"
+				preventPopoverOpen={true}
+				hidePopoverArrow={true}
+				readonly={this.readonly}
+				class="ui5-multi-combobox-tokenizer"
+				disabled={this.disabled}
+				onShowMoreItemsPress={this._showFilteredItems}
+				opener={this.morePopoverOpener}
+				popoverMinWidth={this._inputWidth}
+				onTokenDelete={this._tokenDelete}
+				onBeforeMorePopoverOpen={this.handleBeforeTokenizerPopoverOpen}
+				onFocusOut={this._tokenizerFocusOut}
+				onFocusIn={this._tokenizerFocusIn}
+				onClick={this._click}
+				onPaste={this._handlePaste}
+				onKeyDown={this._onTokenizerKeydown}
+				expanded={this._tokenizerExpanded}
+				open={this.tokenizerOpen}
+			>
+				{this.items.map((item, index) => {
+					if (item.isGroupItem) {
+						return item.items?.map((groupItem, groupItemIndex) => {
+							return groupItem.selected && <Token
+								readonly={this.readonly}
+								class="ui5-multi-combobox-token"
+								data-ui5-id={groupItem._id}
+								part={`token-${groupItemIndex}`}
+								text={groupItem.text}
+							/>;
+						});
+					}
+
+					return item.selected && <Token
+						readonly={this.readonly}
+						class="ui5-multi-combobox-token"
+						data-ui5-id={item._id}
+						part={`token-${index}`}
+						text={item.text}
+					/>;
+				})}
+			</Tokenizer>
+
+			<input
+				id="ui5-multi-combobox-input"
+				value={this.value}
+				inner-input
+				placeholder={this._getPlaceholder}
+				disabled={this.disabled}
+				readonly={this.readonly}
+				value-state={this.valueState}
+				onInput={this._inputLiveChange}
+				onChange={this._inputChange}
+				onKeyDown={this._onkeydown}
+				onClick={this._click}
+				onFocusIn={this.inputFocusIn}
+				onFocusOut={this.inputFocusOut}
+				onPaste={this._handlePaste}
+				role="combobox"
+				aria-haspopup="dialog"
+				aria-expanded={this.open}
+				aria-autocomplete="both"
+				aria-describedby={this.ariaDescribedByText}
+				aria-required={this.required}
+				aria-label={this.ariaLabelText}
+				aria-controls={this.responsivePopoverId}
+				autocomplete="off"
+				data-sap-focus-ref
+			/>
+
+			{this._effectiveShowClearIcon &&
+				<div
+					onClick={this._clear}
+					class="ui5-input-clear-icon-wrapper inputIcon"
+					tabIndex={-1}
+					onMouseDown={this._iconMouseDown}
+				>
+					<Icon tabIndex={-1} class="ui5-input-clear-icon" name={decline} accessible-name={this.clearIconAccessibleName}></Icon>
+				</div>
+			}
+
+			{this.icon &&
+				<slot name="icon"></slot>
+			}
+
+			{!this.readonly &&
+				<Icon name="slim-arrow-down"
+					class="inputIcon"
+					slot="icon"
+					tabIndex={-1}
+					onClick={this.togglePopoverByDropdownIcon}
+					onMouseDown={this._onIconMousedown}
+					onFocusIn={this._forwardFocusToInner}
+					// pressed={this.open}
```

Comment:
I think we did this alternative in other components
"inputIcon--pressed": this.open

Reply:
I could not find any CSS that works with this property. Also, in the original template it was with `?` and bound as a property, so there wasn't even an attribute most probably

Reply:
We switched to a class `inputIcon--pressed` some time ago, in the InputIcon.css.

```css
.inputIcon.inputIcon--pressed {
	background: var(--_ui5_input_icon_pressed_bg);
	box-shadow: var(--_ui5_input_icon_box_shadow);
	border-inline-start: var(--_ui5_select_hover_icon_left_border);
	color: var(--_ui5_input_icon_pressed_color);
}
````

Previously it was like `.inputIcon[pressed]` (which this pressed={this.open} was for), but was not ts-friendly, so we refactored it.